packages/kbn-journeys/journey/journey_ftr_harness.ts

```diff
@@ -160,14 +168,60 @@ export class JourneyFtrHarness {
 
     this.page = await this.context.newPage();
 
+    await this.interceptBrowserRequests(this.page);
+
+    const initializeApmInitListener = async () => {
+      await this.page?.evaluate(() => {
+        const win = window as WindowWithApmContext;
+
+        const attachTraceIdOverrideListener = () => {
+          if (win.traceIdOverrideListenerAttached) {
+            return;
+          }
+          win.traceIdOverrideListenerAttached = true;
+          win.elasticApm!.observe('transaction:start', (tx) => {
```

Comment:
there is a high chance this gets changed before `transaction:end` gets fired. Better would be to do it at that time.